src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/DetectPreviewFeatureTests.Interfaces.cs

```diff
@@ -70,13 +98,48 @@ public interface IProgram
                 public void UnmarkedMethodInMarkedInterface() { }
             }
         }
-
             ";
 
             var test = TestCS(csInput);
             test.ExpectedDiagnostics.Add(VerifyCS.Diagnostic(DetectPreviewFeatureAnalyzer.ImplementsPreviewMethodRule).WithLocation(0).WithArguments("UnmarkedMethodInMarkedInterface", "IProgram.UnmarkedMethodInMarkedInterface", DetectPreviewFeatureAnalyzer.DefaultURL));
             test.ExpectedDiagnostics.Add(VerifyCS.Diagnostic(DetectPreviewFeatureAnalyzer.ImplementsPreviewInterfaceRule).WithLocation(1).WithArguments("Program", "IProgram", DetectPreviewFeatureAnalyzer.DefaultURL));
             await test.RunAsync();
+
+            var vbInput = @" 
+        Imports System
+        Imports System.Runtime.Versioning
+        Module Preview_Feature_Scratch
+            Public Class Program
+                Implements {|#1:IProgram|}
+                Private Shared Sub Main(ByVal args As String())
+                    Dim prog = New Program()
+                End Sub
+
+                Public Sub MarkedMethodInInterface() Implements IProgram.{|#0:MarkedMethodInInterface|}
+                    Throw New NotImplementedException()
+                End Sub
+
+                Public ReadOnly Property Value As String Implements IProgram.{|#2:value|}
+                    {|#3:Get|}
+                        Return """"
+                    End Get
+                End Property
+            End Class
+
+            <RequiresPreviewFeatures>
+            Public Interface Iprogram
+                Sub MarkedMethodInInterface()
+                ReadOnly Property Value() As String 
+            End Interface
+        End Module
```

Comment:
Please make the case-insensitivity test more apparently intentional. Ideally as a separate test that is named to represent this scenario. We could even have a C# test that shows that C# is treated with case sensitivity and then the VB test showing that it's case-insensitive. As it is now, someone in the future might think these were typos and "correct" them. :wink: